.github/workflows/build.yml

```diff
@@ -307,7 +312,7 @@ jobs:
             ################
             # Python
             - name: Install python dependencies
-              run: yarn _requires_python
+              run: pip install jupyter_packaging && yarn _requires_python
```

Comment:
make sure to pin, most of the stuff we use is deprecated and will be removed after `1.0`

Reply:
there's a chicken-.egg problem here: setup.py has to be able to run before dependencies are installed, but setup.py imports items from jupyter_packaging, which is one of its dependencies

the common fix for that seems to be to try/except the jupyter_packaging import and provide dummy values for eg cmdclasses -- however, this change also depends on jupyter_packaging to parse perspective-jupyterlab's version number.   I could just revert that part + so that setup.py continues to own version parsing.

Reply:
note to self: pin the version installed in _wheel_python.js also

Reply:
got this working well enough in setup.py